internal/fuzzbinder/fuzzscraper.go

```diff
@@ -197,31 +151,21 @@ func formatDate(fuzzParameters *FuzzParameters) string {
 	return hyphenDate
 }
 
-// getLogs gets the log-files list of a fuzz-target on a given day.
-func getLogs(fuzzParameters *FuzzParameters, fuzzTarget string) (*storage.BucketHandle, []string, error) {
+// getLogDirInfo gets the bucket and relative path of the directory
+// in which log-files are saved.
+func getLogDirInfo(fuzzParameters *FuzzParameters, fuzzTarget string) (string, string) {
```

Comment:
Is it worth writing some unit tests for it?

Reply:
I added it.